pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -52,386 +53,103 @@ import "./TimelockExecutionHelper.sol";
  * In fact a number of the TimelockAuthorizer's functions may only be called through a scheduled execution so reentrancy
  * is necessary in order to be able to call these.
  */
-contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
-    using Address for address;
+contract TimelockAuthorizer is IAuthorizer, TimelockAuthorizerManagement {
+    // solhint-disable-next-line const-name-snakecase
+    uint256 private constant _max_delay = 2 * (365 days);
```

Comment:
We typically do `_MAX_DELAY` for things such as these. I don't think we ever use lowercase snakecase.